client/app/account/components/AccountInfo.tsx

```diff
@@ -35,16 +35,15 @@ const AccountInfo = () => {
           Create House
         </RouterLink>
       )}
-      {/* TODO: Add functionality for changing password
       <div>
         <h2 className='font-semibold'>Password</h2>
         <RouterLink
-          href='/edit-password'
-          className='text-lg font-normal underline'
+          href='/account/change-password'
+          className='underline underline-offset-4'
```

Comment:
Can you add hover style too? 🙏  Maybe opacity 70 or 80 is good!

Reply:
Right! Added it. 
fa7a2caeced095df603d2e0ae9982658e5c753f0